src/main/java/seedu/address/logic/Messages.java

```diff
@@ -21,6 +21,10 @@ public class Messages {
     public static final String MESSAGE_NONEXISTENT_PERSON =
             "The person '%1$s' does not exist in the system!";
```

Comment:
I like how you formatted the strings for error messages. This makes it easier to insert variable data into the messages, improving the clarity of error communication.